Cargo.toml

```diff
@@ -99,7 +99,7 @@ de_uom = { path = "crates/uom", version = "0.1.0-dev" }
 
 # Other
 ab_glyph = "0.2.21"
-ahash = "0.7.6"
+ahash = "0.8.3"
```

Comment:
`bevy_utils` requires version `^0.7.0` thus I would not upgrade this dependency until Bevy upgrade.